<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositWindow</class>
 <widget class="QWidget" name="DepositWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1137</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">DepositWindow {
 background-color: rgb(207, 248, 255);
}</string>
  </property>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
 background-color: rgb(255, 157, 246);
 color: white;
 border: 1px solid gray;
font: 18pt &quot;Rockwell&quot;;
}

QPushButton:pressed{
 background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
}
</string>
   </property>
   <property name="text">
    <string>Go back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_btn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>310</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
 background-color: rgb(227, 167, 255);
 color: white;
 border: 1px solid gray;
font: 18pt &quot;Rockwell&quot;;
}

QPushButton:pressed{
 background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(245, 224, 176, 255), stop:0.09 rgba(246, 189, 237, 255), stop:0.14 rgba(194, 207, 246, 255), stop:0.19 rgba(184, 160, 168, 255), stop:0.25 rgba(171, 186, 248, 255), stop:0.32 rgba(243, 248, 224, 255), stop:0.385 rgba(249, 162, 183, 255), stop:0.47 rgba(100, 115, 124, 255), stop:0.58 rgba(251, 205, 202, 255), stop:0.65 rgba(170, 128, 185, 255), stop:0.75 rgba(252, 222, 204, 255), stop:0.805 rgba(206, 122, 218, 255), stop:0.86 rgba(254, 223, 175, 255), stop:0.91 rgba(254, 236, 244, 255), stop:1 rgba(255, 191, 221, 255));
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate_btn">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>310</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
 background-color: rgb(175, 174, 255);
 color: white;
 border: 1px solid gray;
 font: 18pt &quot;Rockwell&quot;;
}

QPushButton:pressed{
 background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(9, 41, 4, 255), stop:0.085 rgba(2, 79, 0, 255), stop:0.19 rgba(50, 147, 22, 255), stop:0.275 rgba(236, 191, 49, 255), stop:0.39 rgba(243, 61, 34, 255), stop:0.555 rgba(135, 81, 60, 255), stop:0.667 rgba(121, 75, 255, 255), stop:0.825 rgba(164, 255, 244, 255), stop:0.885 rgba(104, 222, 71, 255), stop:1 rgba(93, 128, 0, 255));
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="deposit_amount_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>                     Deposit amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="deposit_term_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>                       Deposit term</string>
   </property>
  </widget>
  <widget class="QLabel" name="interest_rate_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>                        Interest rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="tax_rate_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>                            Tax rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="periodicity_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>                Periodicity of payments</string>
   </property>
  </widget>
  <widget class="QLabel" name="monthly_replenishment_lbl">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>180</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>              Monthly replenishment</string>
   </property>
  </widget>
  <widget class="QLabel" name="monthly_withdrawal_lbl">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>                 Monthly withdrawal</string>
   </property>
  </widget>
  <widget class="QLabel" name="accrued_interest_lbl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>                     Accrued interest</string>
   </property>
  </widget>
  <widget class="QLabel" name="deposit_by_the_end_lbl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>480</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>Deposit amount by the end of the term</string>
   </property>
  </widget>
  <widget class="QLabel" name="tax_amount_lbl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>                         Tax amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deposit_by_the_end_lineEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>480</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accrued_interest_lineEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>360</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tax_amount_lineEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>420</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deposit_amount_lineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deposit_term_lineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="interest_rate_lineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="periodicity_groupBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(189, 118, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="period_monthly_radio_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Rockwell&quot;;
 background-color: rgb(189, 118, 255);</string>
    </property>
    <property name="text">
     <string>monthly</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="period_yearly_radio_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Rockwell&quot;;
 background-color: rgb(189, 118, 255);</string>
    </property>
    <property name="text">
     <string>yearly</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="tax_rate_lineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="capitalization_checkBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>120</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
 background-color: rgb(189, 118, 255);</string>
   </property>
   <property name="text">
    <string>          Capitalization of interest</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="monthly_withdrawal_lineEdit">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>240</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="monthly_replenishment_lineEdit">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>180</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="percent_2_lbl">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>240</y>
     <width>21</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="percent_1_lbl">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>21</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="months_lbl">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Rockwell&quot;;
background-color: rgb(202, 176, 255);</string>
   </property>
   <property name="text">
    <string>months</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
